build/WinObjC.Frameworks.Core/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "dependencies": {
+    "NuGet.Build.Packaging": "0.2.0"
```

Comment:
! DANGER !

I don't think we can build with 0.2.0. Are we specifying conflicting versions in the project versus in the project.json?